src/Chapter01/Listing01.04.BreakingApartHelloWorld.cs

```diff
@@ -1,12 +1,13 @@
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter01.Listing01_04
 {
     #region INCLUDE
-    public class Program//Class definition
+    public class Program                // BEGIN Class definition
```

Comment:
We should check how this formats in the manuscript, I wonder what this might look like

Reply:
I assume it looks like a comment. I'm not sure I understand what you are asking.